src/components/top-features.js

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+
+ const TopFeatures = () => (
+  <section>
+    <div class="cmp-percentage-card-grid">
+      <div class="cmp-percentage-card">
```

Comment:
This will need to have an unordered list structure as well.

And thinking about it, this might be better served as an ordered list, because we are listing them in a rank order by percentage. That means switching the second list below to an `ol` as well, but then adding the `start` attribute to signify that we’re starting at #4. But that also means we probably need number in the top 3, which is not designed that way. 😕 

I don’t know. Let me know what you think about that shift. Does this data make better sense as an ordered list or an unordered list, and if ordered, how should we display that information.

Reply:
I think it makes more sense for it to be ordered. Maybe we can use the Yellowtail style for the numbers here, too? I agree that the top three will need numbers as well. Using the blue Yellowtail style definitely has potential to make it look a bit busy, though. What do you think?

Reply:
Yeah. I think the numbers here probably should be subtle. I have an idea, but we’ll have to see if it works.

Reply:
Let use the pink `ol` style from the standard ordered list.